app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,5 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{rental-listing rental=rentalUnit goToShowRoute="goToShowRoute"}}
```

Comment:
Added an action to make the component clickable per recommendation here:
https://github.com/emberjs/guides/pull/1443#discussion_r64994911

Wondering if this is out of scope for a tutorial on sub-routes.  Not sure I want to encourage writing actions this way with closure actions and the route-action helper becoming more prominent. But I don't want to introduce a controller just for the sake of a closure action.

Anyone have thoughts on a faster way to do this that doesn't take much explaining for a beginner's tutorial?  Maybe just wrap component in a div and add the action to that?